webapp/src/selectors/index.ts

```diff
@@ -13,17 +13,21 @@ const getPluginState = (state) => state['plugins-' + PluginId] || {};
 
 export const getPluginServerRoute = (state) => {
     const config = getConfig(state);
-
     let basePath = '';
+    let url;
```

Comment:
If we store in a variable only the part that changes, we don't need this to be a variable nor to use twice `new URL(...)`, let me give you an example!

```typescript
const baseUrl = config?.configSiteURL ? config.SiteUrl : window.location.href;
const basepath = new URL(baseUrl).pathname;
```